app/server/api/metrics_api.py

```diff
@@ -84,6 +84,7 @@ def get(self):
         return make_response(jsonify(response_object)), 200
 
 class FiltersApi(MethodView):
+    @multi_org
```

Comment:
Filter needs to support multi org or alternatively be disabled on the frontend. @michielderoos what do you think?

Reply:
This is reasonable!